Makefile

```diff
@@ -22,7 +22,7 @@ clean-indexer:
 .PHONY: test
 
 test:
-	go test -race -v ./...
+	go clean -testcache && go test -race -v ./...
```

Comment:
FWIW, the idiomatic way of doing this is `go test -race -v -count=1 ./...`. It's weird, I know, but Go developers will immediately understand what you are trying to do 🙂.

That said, I'm not sure we always want to bypass the test cache, so I wouldn't force the make target to do it.